packages/react-form-with-constraints-bootstrap/scss/bootstrap.scss

```diff
@@ -0,0 +1,14 @@
+// Bootstrap likes to hide feedbacks, we don't like it, especially react-form-with-constraints-tools's DisplayFields
+// [.invalid-feedback doesn't show for .input-group with an invalid control](https://github.com/twbs/bootstrap/issues/23454)
+.valid-feedback,
+.invalid-feedback {
+  // stylelint-disable-next-line declaration-no-important
+  display: block !important; // [Is there an opposite to display:none?](https://stackoverflow.com/a/37289654/990356)
+}
+
+// https://github.com/twbs/bootstrap/blob/v4.1.3/scss/_variables.scss#L572
+// $form-feedback-valid-color => $success => $green => #28a745
+// $form-feedback-invalid-color => $danger => $red => #dc3545
+
+$form-feedback-warning-color: map-get($theme-colors, 'warning') !default; // $yellow => #ffc107
+$form-feedback-info-color: map-get($theme-colors, 'info') !default; // $cyan => #17a2b8
```

Comment:
Defining 2 variables won't do any magic.

You should call form-validation-state mixin from Bootstrap otherwise nothing will happen.

See https://getbootstrap.com/docs/5.0/forms/validation/#mixins & https://github.com/twbs/bootstrap/blob/v5.0.1/scss/mixins/_forms.scss#L18